github/code-scanning.go

```diff
@@ -27,20 +72,50 @@ type Tool struct {
 	Version *string `json:"version,omitempty"`
 }
 
+// DismissedBy provides details about the person who dismissed the Alert.
```

Comment:
Let's please delete `struct DismissedBy` entirely here and re-use the already-existing `User` struct instead.

Reply:
Good catch! Re-used `User` struct